snuba/query/snql/parser.py

```diff
@@ -1305,11 +1307,6 @@ def validate_entities_with_query(
                 f"validation failed for entity {query.get_from_clause().key.value}: {e}",
                 should_report=e.should_report,
             )
-        except InvalidExpressionException as e:
```

Comment:
This is unrelated to the change, but I fixed it anyways. This check is redundant since `InvalidExpressionException` is a subclass of `InvalidQueryException`.